doc/manual/src/command-ref/nix-copy-closure.md

```diff
@@ -14,62 +14,51 @@
 
 # Description
 
-`nix-copy-closure` gives you an easy and efficient way to exchange
-software between machines.  Given one or more Nix store _paths_ on the
-local machine, `nix-copy-closure` computes the closure of those paths
-(i.e. all their dependencies in the Nix store), and copies all paths
-in the closure to the remote machine via the `ssh` (Secure Shell)
-command.  With the `--from` option, the direction is reversed: the
-closure of _paths_ on a remote machine is copied to the Nix store on
-the local machine.
-
-This command is efficient because it only sends the store paths
-that are missing on the target machine.
-
-Since `nix-copy-closure` calls `ssh`, you may be asked to type in the
-appropriate password or passphrase.  In fact, you may be asked _twice_
-because `nix-copy-closure` currently connects twice to the remote
-machine, first to get the set of paths missing on the target machine,
-and second to send the dump of those paths.  When using public key
-authentication, you can avoid typing the passphrase with `ssh-agent`.
+Given _paths_ accesible from the source machine, `nix-copy-closure` computes the [closure](@docroot@/glossary.md#gloss-closure) of those paths (i.e. all their dependencies in the Nix store), and copies [store objects](@docroot@/glossary.md#gloss-store-object) in the closure to the target machine via SSH.
+It doesn’t copy store objects that are already present on the target machine.
+
+> **Note**
+>
+> While the Nix store to use on the client can be specified on the command line with the [`--store`](@docroot@/command-ref/conf-file.md#conf-store) option, the Nix store to be accessed on the remote machine can only be [configured statically](@docroot@/command-ref/conf-file.md#configuration-file) on that remote machine.
```

Comment:
What is the client? The `nix-copy-closure` command?
I guess it might make sense to borrow that term from SSH, but that precisely not what this store is about.

Reply:
```suggestion
> The default Nix store (used when perhaps either `--from` or `--to` is unset) can be specified on the command line with the [`--store`](@docroot@/command-ref/conf-file.md#conf-store) option, but the Nix store to be accessed on the remote machine(s) can only be [configured statically](@docroot@/command-ref/conf-file.md#configuration-file) on that remote machine.
```